js/game.js

```diff
@@ -1,4 +1,4 @@
-const app3=new Vue({
+const appgame=new Vue({
```

Comment:
i dont get this?

Reply:
@gauthamzz 

The variable name **app3** was the same in game.js and leaderBoard.js

So I changed the name in game.js because it was an error.